Vagrantfile

```diff
@@ -6,11 +6,12 @@ BOX_CPUS = ENV["BOX_CPUS"] || "1"
 BOX_MEMORY = ENV["BOX_MEMORY"] || "1024"
 DOKKU_DOMAIN = ENV["DOKKU_DOMAIN"] || "dokku.me"
 DOKKU_IP = ENV["DOKKU_IP"] || "10.0.0.2"
+DOKKU_USER = ENV["DOKKU_USER"] || "dokku"
 FORWARDED_PORT = (ENV["FORWARDED_PORT"] || '8080').to_i
 PREBUILT_STACK_URL = File.exist?("#{File.dirname(__FILE__)}/stack.tgz") ? 'file:///root/dokku/stack.tgz' : nil
 PUBLIC_KEY_PATH = "#{Dir.home}/.ssh/id_rsa.pub"
 
-make_cmd = "DEBIAN_FRONTEND=noninteractive make -e install"
+make_cmd = "DEBIAN_FRONTEND=noninteractive make -e install DOKKU_USER=#{DOKKU_USER}"
```

Comment:
I expect this will need to go in front of the `make` command similar to `DEBIAN_FRONTEND`. If it indeed works this way, then you can downgrade this to a style nitpick. :smile: